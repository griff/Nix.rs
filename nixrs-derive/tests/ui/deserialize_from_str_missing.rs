use nixrs_derive::NixDeserialize;

#[derive(NixDeserialize)]
#[nix(from_str)]
pub struct Test;

fn main() {}
