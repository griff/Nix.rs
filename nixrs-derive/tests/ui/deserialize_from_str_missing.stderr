error[E0277]: the trait bound `Test: FromStr` is not satisfied
 --> tests/ui/deserialize_from_str_missing.rs
  |
  | #[nix(from_str)]
  |       ^^^^^^^^ the trait `FromStr` is not implemented for `Test`
  |
  = help: the following other types implement trait `FromStr`:
            IpAddr
            Ipv4Addr
            Ipv6Addr
            NonZero<i128>
            NonZero<i16>
            NonZero<i32>
            NonZero<i64>
            NonZero<i8>
          and $N others
